### PR TITLE
feat: like, watchlater & explore page

### PR DESCRIPTION
**Pull request by PuneetP16**
Features Added:
1. Add or remove from like or watchlist toggle
2. Explore page listing all the videos.

image

image

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Code structure is good and clean.
1. variable naming is great
2. you can create some components for repeating things

otherwise its is good and ui is great
you can go ahead

File: src/components/Cards/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -21,6 +38,13 @@ export const VideoCard = ({ video }) => {
...
+	const { axiosRequest } = useAxios();
+	const { isAuth } = useAuth();
+
+	const navigateToLogin = () => {
+		if (!isAuth) navigate("/login");
+	};
+
 	console.log({
 		_id,
 		title,
```
Comment (saurabhchirde) [inline src/components/Cards/VideoCard/VideoCard.jsx:50]: remove console log

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue, as you have written your code immaculately. Just a suggestion if you could use the absolute path, otherwise it's all good.
You can proceed to merge.

File: src/components/Cards/VideoCard/VideoCard.jsx (hunk 1 above)
Comment (PuneetP16) [reply src/components/Cards/VideoCard/VideoCard.jsx:50]: Thanks for the suggestion will do that

**Pull request merged**